docs/changes/visualization/11790.feature.rst

```diff
@@ -0,0 +1 @@
+Added a feature so that SphericalCircle will accept center parameter as an object.
```

Comment:
It think it would be better if the change log entry would mention `SkyCoord` explicitly.